lib/galaxy/tools/linters/inputs.py

```diff
@@ -24,34 +24,50 @@ def lint_inputs(tool_xml, lint_ctx):
         if param_type == "data":
             if "format" not in param_attrib:
                 lint_ctx.warn("Param input [%s] with no format specified - 'data' format will be assumed.", param_name)
+
```

Comment:
No, this just synced all of planemo's libs. I think that is fine - let me know though.